README.md

```diff
@@ -88,6 +89,15 @@ This will start the process of "ejecting" from Create React Native App's build s
 * **One Build Tool**: If you just want to get started with React Native, you shouldn't need to install Xcode, Android Studio, NDKs, or mess with environment variables.
 * **No Lock-In**: You can always "eject" to your own build setup if you need to write custom native code or modify how your app is built.
 
+## Known Issues
+
+There are some issues that have been identified by us and are documented here:
+
+* **QR code can't be scanned when using Webstorm's built-in terminal** [#49](https://github.com/react-community/create-react-native-app/issues/49)
```

Comment:
WebStorm has a `Console Colors` Tab as well. Should I try different settings and then document what works, and what doesn't? Because, AFAIK, the themes offered online for WebStorm and editors alike, don't really change the console colors as such. 

So, it would be better if we added settings for the Console Colors rather than asking them to change their entire color scheme (which technically, wouldn't change anything.) 

What do you reckon?